backend/src/main/java/gov/cdc/usds/simplereport/service/email/EmailProviderWrapper.java

```diff
@@ -0,0 +1,8 @@
+package gov.cdc.usds.simplereport.service.email;
+
+import com.sendgrid.helpers.mail.Mail;
+import java.io.IOException;
+
+public interface EmailProviderWrapper {
```

Comment:
This isn't a wrapper, either—just call it `EmailProvider` and then your implementation classes can be `LiveEmailProvider` and something else. Or you can not have a separate provider and make the service behavior conditional in some other way, which seems simpler to me.